src/utils/KML.js

```diff
@@ -237,7 +237,17 @@ const writeFeatures = (layer, featureProjection) => {
 
     const olStyle = new Style(newStyle);
     clone.setStyle(olStyle);
-    exportFeatures.push(clone);
+
+    if (
+      clone.getGeometry() instanceof Point &&
+      olStyle.getText() &&
+      !olStyle.getText().getText().length
```

Comment:
```suggestion
      !olStyle.getText().getText()
```
no need to use length

Reply:
self commited